frontends/p4/tableKeyNames.cpp

```diff
@@ -68,10 +86,17 @@ class KeyNameGenerator : public Inspector {
             error(expression); }
     }
 
+    /** Compute a name annotation for @expression.  Eg. `@name("16w4")` for
```

Comment:
I expect that the result would be 4 instead of 16w4.
I don't think that the control-plane (the one that consumes these annotations) supports widths.